src/reconcile/rec_col.c

```diff
@@ -875,7 +875,7 @@ __wt_rec_col_fix(
             /* We shouldn't ever get appends during salvage. */
             WT_ASSERT(session, salvage == NULL);
 
-            WT_ERR(__wt_rec_upd_select(session, r, ins, NULL, NULL, &upd_select));
+            WT_ERR(__wt_rec_upd_select(session, r, WT_RECNO_OOB, ins, NULL, NULL, &upd_select));
```

Comment:
I believe in this case instead of recno the `WT_RECNO_OOB` is passed because internally it is not used. If it is correct. Add a comment to explain it. I feel it is better to pass the recno as it is available to avoid confusion about when to pass.

Reply:
Fix in 1793